src/libraries/Microsoft.PowerFx.Core/Parser/TexlParser.cs

```diff
@@ -56,8 +57,15 @@ private TexlParser(IReadOnlyList<Token> tokens, Flags flags)
             _depth = 0;
             _curs = new TokenCursor(tokens);
             _flagsMode = new Stack<Flags>();
-            _flagsMode.Push(flags);
-        }
+            _flagsMode.Push(flags);
+            _locale = CultureInfo.CurrentCulture;
```

Comment:
> CultureInfo.CurrentCulture; [](http://example.com/codeflow?start=21&length=28)

we're trying to remove calls to CultureInfo.CurrentCulture :) 

Parser is conveniently internal - so we can change the signature. Can we make the param required and avoid a call to CultureInfo.CurrentCulture;